static/keycloak.json

```diff
@@ -0,0 +1,8 @@
+{
+  "realm": "fcf0kpqr",
+  "auth-server-url": "https://dev.oidc.gov.bc.ca/auth",
```

Comment:
But this is the dev environment. Need to figure out how to substitute Test and Prod kc files for running in those environments.